spec/lib/merge_supporters_spec.rb

```diff
@@ -259,11 +259,11 @@
   end
 
   describe '.merge_by_id_groups' do
-    it 'merges the supporters from the id groups' do
-      supporter_1 = np.supporters.create!(name: 'Penelope Schultz')
-      supporter_2 = np.supporters.create!(name: 'Cacau Borges')
-      result = MergeSupporters.merge_by_id_groups(np.id, [[supporter_1.id, supporter_2.id]], nil)
+    let(:supporter_1) { np.supporters.create!(name: 'Penelope Schultz') }
+    let(:supporter_2) { np.supporters.create!(name: 'Cacau Borges') }
+    let(:result) { MergeSupporters.merge_by_id_groups(np.id, [[supporter_1.id, supporter_2.id]], nil) }
```

Comment:
There's positives and negatives to using `let`. My general feeling is we should use less of it because it's hard to keep track of things. But I'm cool with moving it in this case.